types/echarts/index.d.ts

```diff
@@ -1590,8 +1590,8 @@ declare namespace echarts {
                     rotate?: number | undefined;
                     margin?: number | undefined;
                     formatter?: string | Function | undefined;
-                    showMinLabel?: boolean | undefined;
-                    showMaxLabel?: boolean | undefined;
+                    showMinLabel?: boolean | null | undefined;
+                    showMaxLabel?: boolean | null | undefined;
```

Comment:
`null` should not be required here since the optional mark `?` in the type already include the null value.

Reply:
Typescript will check failed If I want to set showMinLabel to null.